### ISSUE TITLE
Location for home pages without a 'visual current home link' a fail for 2.4.8: Location?

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1750]
Just for the record, but a home page often doesn't have a location indicator like breadcrumbs or alike.
Also the menu is often to other pages but not for the home page itself, where the logo serves the 'home link'.
(for the sake of site maps should not be used as a fall back... https://github.com/w3c/wcag/issues/1696)

So should a home page be excepted from this SC?

Examples from Deque demo pages:

Without HOME link in menu (FAIL?): https://dequeuniversity.com/demo/dream
With HOME link in menu (PASS!): http://webtestingcourse.dequecloud.com/

**Comment by jake-abma** [comment]
PS, why are 'Titles' not mentioned in the Understanding, also as a Technique, as Title Understanding mentioned:

> Titles identify the current location without requiring users to read or interpret page content.

See also related issue 1763

**Comment by alastc** [comment]
> a home page often doesn't have a location indicator like breadcrumbs or alike.

I think the short answer is: At AAA, the homepage could have a location indicator.

**Comment by jake-abma** [comment]
Good catch, but also you agree then that without it, it fails... as in, only a logo with link to the page itself = FAIL

**Comment by alastc** [comment]
Assuming that the logo & link are not marked in anyway differently from the same logo/link on other pages, I would agree that fails.
